lib/Db/Row2.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace OCA\Tables\Db;
+
+use JsonSerializable;
+use OCA\Tables\ResponseDefinitions;
+
+/**
+ * @psalm-import-type TablesRow from ResponseDefinitions
+ */
+class Row2 implements JsonSerializable {
+	private ?int $id = null;
+	private ?int $tableId = null;
+	private ?string $createdBy = null;
+	private ?string $createdAt = null;
+	private ?string $lastEditBy = null;
+	private ?string $lastEditAt = null;
```

Comment:
Those seem not nullable in the schema at https://github.com/nextcloud/tables/pull/749/files?diff=split#diff-ece721377378e6d727a89e8d2f9a7c9ed316ee1971e781a69c342aea6a8927f5R75-R78 so would probably be good to also let that be the case in the row entity